Packs/CommonScripts/Scripts/IfThenElse/IfThenElse.js

```diff
@@ -21,6 +21,15 @@ function getValue(args, condition, options, lhs) {
     throw `Invalid condition: ${condition}`;
 }
 
+function getValueB(args, second_conditionB, optionsB, lhsB) {
```

Comment:
@melamedbn You don't need to duplicate the function and hard code the array inside the for, just send it as a variable to the function and also make the